src/core/ext/filters/client_channel/lb_policy_factory.h

```diff
@@ -66,9 +66,18 @@ typedef struct grpc_lb_addresses {
 grpc_lb_addresses *grpc_lb_addresses_create(
     size_t num_addresses, const grpc_lb_user_data_vtable *user_data_vtable);
 
+/** Deep copy the address from \a src to \a dest. */
+void grpc_lb_address_copy(grpc_lb_address* dest, const grpc_lb_address* src,  
```

Comment:
That's fine.  (If you were going to have a separate function for copying only the backend addresses, I would say to keep the helper function, to avoid duplication between the two "public" methods.  But if you're just going to add a parameter to `grpc_lb_addresses_copy()`, then there's no need for that.)